internal/pkg/cli/pipeline_status.go

```diff
@@ -4,72 +4,211 @@
 package cli
 
 import (
+	"errors"
 	"fmt"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/codepipeline"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli/selector"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/config"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/deploy/cloudformation/stack"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/describe"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/manifest"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/workspace"
+	"io"
 
 	"github.com/spf13/cobra"
 )
 
+const (
+	pipelineStatusAppNamePrompt          = "Which application's pipeline's status would you like to show?"
```

Comment:
maybe just `Which application's pipeline status...`? I'm not an expert in rewording though.

Reply:
I do think that sounds better. May not be quite as accurate, though--sounds more like all pipelines/pipeline show? What do other people think?